ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -848,17 +1149,20 @@ const triggerSet: TriggerSet<Data> = {
         } else if (id === headmarkers.dot2) {
           data.diveFromGraceNum[matches.target] = 2;
           if (matches.target === data.me)
-            return output.num2!();
+            return output.stackNorthNum!({ num: output.num2!() });
         } else if (id === headmarkers.dot3) {
           data.diveFromGraceNum[matches.target] = 3;
           if (matches.target === data.me)
-            return output.num3!();
+            return output.stackNorthNum!({ num: output.num3!() });
         }
       },
       outputStrings: {
         num1: Outputs.num1,
         num2: Outputs.num2,
         num3: Outputs.num3,
+        stackNorthNum: {
+          en: '${num}, Stack North',
```

Comment:
I did mention the prepositioning, but I think this call is fine even with a prepositioning strat for 3s.  It's a reminder that they need to make their way north.